target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -33,8 +38,21 @@ contract EntropyTest is Test, EntropyTestUtils {
     uint128 MAX_UINT128 = 2 ** 128 - 1;
     bytes32 ALL_ZEROS = bytes32(uint256(0));
 
+    address public owner = address(8);
+    address public admin = address(9);
+    address public admin2 = address(10);
+
     function setUp() public {
-        random = new Entropy(pythFeeInWei, provider1, false);
+        EntropyUpgradable _random = new EntropyUpgradable();
+        // deploy proxy contract and point it to implementation
+        proxy = new UUPSProxy(address(_random), "");
+        // wrap in ABI to support easier calls
+        random = EntropyUpgradable(address(proxy));
```

Comment:
Why you are not using ERC1967 directly and created UUPSProxy? :think:

Reply:
lol, i copied things from a place and missed it somehow. thanks !!!